docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
Do these three `docker/*` files belong in this PR?  Looks like they were added by mistake